core/arch/arm/plat-ti/a9_plat_init.S

```diff
@@ -104,6 +104,9 @@ after_resume:
 	mov	r0, r4
 	bl	init_sec_mon
 
+	bl	main_init_gic
```

Comment:
Yes, our supported A9 SoCs are all single core. We could check core_pos() and call secondary_init_gic() if needed, but we could not test that and it would never get used.

Reply:
OK, understood.